helm-charts/splunk-otel-collector/values.schema.json

```diff
@@ -532,6 +532,28 @@
             }
           }
         },
+        "journald": {
+          "description": "Configuration to collect journald logs",
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "enabled": {
+              "type": "boolean"
+            },
+            "directory": {
+              "type": "string"
+            },
+            "index": {
+              "type": "string"
+            },
+            "units": {
+              "type": "array",
+              "items": {
+                "type": "object"
```

Comment:
Hey @luckyj5 , I'm sorry to keep bugging you with these small asks, but I just found this.

We need to specify schema for this object since it can only have two keys: `name` and `priority`.

I believe we can merge the PR after that